test/scripts/e2e_subs/box-search.sh

```diff
@@ -136,10 +137,20 @@ NAME=$(${gcmd} app box info --app-id "$APPID" --name "$BOX_NAME" | grep Name | t
 VALUE=$(${gcmd} app box info --app-id "$APPID" --name "$BOX_NAME" | grep Value | tr -s ' ' | cut -d" " -f2-)
 [ "$VALUE" = str:$GREAT_VALUE ]
 
-sleep 15                        # again, app box list only hits DB
 
-# Confirm that we can still get the list of boxes
-BOX_LIST=$(${gcmd} app box list --app-id "$APPID")
+# Confirm that we can still get the list of boxes (need to keep asking
+# until the returned results are for $ROUND)
+retry=0
+while [ $retry -lt 10 ]; do
+    BOX_LIST=$(${gcmd} app box list --app-id "$APPID")
+    ROUND=$(echo "$BOX_LIST" | awk '/Round: / {print $2}')
+    if [[ "$COMMIT" == "$ROUND" ]]; then
```

Comment:
Should work as is, but maybe:

```suggestion
    if [[ "$COMMIT" <= "$ROUND" ]]; then
```